querydsl-sql/src/test/java/com/querydsl/sql/ConfigurationTest.java

```diff
@@ -68,9 +74,16 @@ public void get_schema() {
         assertEquals("pub", configuration.getOverride(new SchemaAndTable("public", "")).getSchema());
         assertEquals("emp", configuration.getOverride(new SchemaAndTable("", "employee")).getTable());
         assertEquals("employees", configuration.getOverride(new SchemaAndTable("public", "employee")).getTable());
-//        assertEquals("pub", configuration.getSchema("public"));
-//        assertEquals("emp", configuration.getTable("", "employee"));
-//        assertEquals("employees", configuration.getTable("public", "employee"));
+
+        configuration.setDynamicNameMapping(new ChangeCaseNameMapping(LetterCase.UPPER));
+        String notDirectOverriden = "notDirectOverriden";
+        assertEquals(notDirectOverriden.toUpperCase(),
```

Comment:
I actually meant the test code in this particular comment.
But that is also a good point.